src/app/api/greenbean/route.ts

```diff
@@ -0,0 +1,75 @@
+import publicClient from "@/lib/publicClient";
+import { NextRequest, NextResponse } from "next/server";
+import ABI from "../../../abis/GREEN_BEAN_ABI.json";
```

Comment:
this is amazing. i don't know you can load API like this before, thanks!

Reply:
With pleasure bro!